src/sass/_list-view.scss

```diff
@@ -0,0 +1,201 @@
+@use '_variables.scss' as *;
+
+.list-container {
```

Comment:
We can actually simplify this quite a bit by leveraging what is in fonts-general-styles & changing the first line of text to a `h2`, and adding the class of `.second-line` to the font file.

```
.list-container {
	display: grid;
	place-items: center;
	.add-button {
		position: relative;
		color: $brown;
		background-color: $yellow;
		&:hover,
		&:focus {
			border-color: $purple;
			background-color: $purple;
			color: $light-green;
		}
		&::before {
			position: absolute;
			bottom: -37px;
			left: -36px;
			content: url('/img/before.svg');
		}
		&::after {
			position: absolute;
			top: -9px;
			right: -38px;
			content: url('/img/after.svg');
		}
	}
}
```